contrib/question_converters/canvas/quiz2pl.py

```diff
@@ -118,6 +118,20 @@ def image_file_extension(content_type: str) -> str:
             return content_type.split("/")[1]
 
 
+FORMULA_IMG_RE = re.compile(
+    r'<img[^>]*data-equation-content="([^"]+)"([^=">]*="([^"]*)")*[^>="]*>',
+    re.DOTALL | re.IGNORECASE,
+)
+
+
+def handle_formulas(text: str) -> str:
+    def replace_formula(match: re.Match[str]) -> str:
+        formula = match.group(1).replace("$", r"\$")
```

Comment:
nit: In my understanding, this strips all other properties other than `data-equation-content`, and for that content, it escapes the dollar sign.

Can you add a comment explaining that since the regex may be hard to read.

Reply:
I added comments for both concerns.